packages/sitecore-jss/src/layout/utils.test.ts

```diff
@@ -53,4 +54,116 @@ describe('sitecore-jss layout utils', () => {
       expect((result[0] as ComponentRendering).componentName).to.be.equal('placed');
     });
   });
+
+  describe('isFieldValueEmpty', () => {
+    it('should return true if passed parameter is not present', () => {
+      const field = {};
+      const result = isFieldValueEmpty(field);
+      expect(result).to.be.true;
+    });
+
+    it('should return true if src is empty for GenericFieldValue', () => {
```

Comment:
Let's name these test cases a bit more explicitly. This one is related to _ImageField_.
May be you can structure _describe_ blocks by field type?
e.g.
```js
describe('Image' ...)
describe('Link' ...)
...
```